t/cobrand/cheshireeast.t

```diff
@@ -167,7 +167,7 @@ FixMyStreet::override_config {
     ok $mech->host("cheshireeast.fixmystreet.com"), "change host to cheshireeast";
     $mech->log_in_ok($staff_user->email);
 
-    $report->comments->delete_all;
+    ($report) = $mech->create_problems_for_body( 1, $body->id, 'Test', { category => 'Zebra Crossing' } );
```

Comment:
This is to fix a broken test when merged with the assignee stuff in `commercial-staging` as those tests delete all reports before we reach this point.